src/sentry/incidents/logic.py

```diff
@@ -516,8 +519,12 @@ def create_alert_rule(
     monitor_type: AlertRuleMonitorTypeInt = AlertRuleMonitorTypeInt.CONTINUOUS,
     activation_condition: AlertRuleActivationConditionType | None = None,
     description: str | None = None,
+    sensitivity: AlertRuleSensitivity | None = None,
+    seasonality: AlertRuleSeasonality | None = None,
+    detection_type: AlertRuleDetectionType = AlertRuleDetectionType.STATIC,
```

Comment:
I was just looking at this as a part of my backfill PR - I think we might infer the `detection_type` based on the presence of other fields rather than passing it in.  If there is a `comparison_delta` that means it's a percent type, and if there are `sensitivity` and `seasonality` values that means it's a dynamic type, otherwise it's static. Lmk if you disagree or think it should be handled higher up